knexfile.js

```diff
@@ -0,0 +1,35 @@
+// Update with your config settings.
+require('dotenv').config(); 
+const pg = require('pg');
+pg.defaults.ssl = true;
+
+module.exports = {
+
+  development: {
+    client: 'sqlite3',
+    connection: {
+      filename: './data/foodieFun.db3'
+    },
+    useNullAsDefault: true,
+    migrations: {
+      directory: './data/migrations',
+    },
+    seeds: {
+      directory: './data/seeds',
+    },
+  },
+
+  production: {
+    client: 'pg',
+    connection: process.env.DATABASE_URL,
```

Comment:
Committing up your database url largely defeats the purpose of abstracting it away!